### PR TITLE
Fix relative time test

### PR DESCRIPTION
**Pull request by tyiu**

### CODE REVIEW

File: damusTests/TimeAgoTests.swift [hunk 1]
```diff
@@ -18,11 +18,11 @@ final class TimeAgoTests: XCTestCase {
         XCTAssertEqual(time_ago_since(Date.now.addingTimeInterval(-2), calendar), "now")
         XCTAssertEqual(time_ago_since(Date.now.addingTimeInterval(-3), calendar), "3s")
         XCTAssertEqual(time_ago_since(Date.now.addingTimeInterval(-59), calendar), "59s")
-        XCTAssertEqual(time_ago_since(Date.now.addingTimeInterval(-60), calendar), "1min")
-        XCTAssertEqual(time_ago_since(Date.now.addingTimeInterval(-3599), calendar), "59min")
-        XCTAssertEqual(time_ago_since(Date.now.addingTimeInterval(-3600), calendar), "1h")
-        XCTAssertEqual(time_ago_since(Date.now.addingTimeInterval(-86399), calendar), "23h")
-        XCTAssertEqual(time_ago_since(Date.now.addingTimeInterval(-86400), calendar), "1d")
+        XCTAssertEqual(time_ago_since(Date.now.addingTimeInterval(-60), calendar), "1m")
```
Comment (tyiu) [inline damusTests/TimeAgoTests.swift:21]: `min` is the abbreviation for minutes in `en-CA`, `m` is the abbreviation for minutes in `en-US`. This test anchors to `en-US`.

File: damusTests/TimeAgoTests.swift [hunk 2]
```diff
@@ -18,11 +18,11 @@ final class TimeAgoTests: XCTestCase {
         XCTAssertEqual(time_ago_since(Date.now.addingTimeInterval(-2), calendar), "now")
         XCTAssertEqual(time_ago_since(Date.now.addingTimeInterval(-3), calendar), "3s")
         XCTAssertEqual(time_ago_since(Date.now.addingTimeInterval(-59), calendar), "59s")
-        XCTAssertEqual(time_ago_since(Date.now.addingTimeInterval(-60), calendar), "1min")
-        XCTAssertEqual(time_ago_since(Date.now.addingTimeInterval(-3599), calendar), "59min")
```
Comment (tyiu) [inline damusTests/TimeAgoTests.swift:22]: Moved away from using seconds for time at the hour and beyond mark to more easily reason about and avoid needing to do manual calculations when writing these assertions.

**Comment by jb55** [comment]
would be nice if we could run these in the PR somehow, not sure if possible.

**Comment by tyiu** [comment]
> would be nice if we could run these in the PR somehow, not sure if possible.

@jb55 Seems like CircleCI has a free tier for open-source projects for hosted continuous integration. It has macOS VM and GitHub integration support. I haven't used it personally but seems popular.

**Comment by jb55** [comment]
thx!

**Pull request closed without merging**